src/tests/examples/output_compare_test.py

```diff
@@ -102,16 +107,43 @@ def compare_directory(result_dir, reference_dir):
                 compare_directory(x, reference_dir / x.name)
         else:
 
-            if x.name != 'id-daily.txt' :
+            uncompared_file_name = ['id-daily.txt', 'id-hourly.txt']
```

Comment:
variable **dir_list**, at beginning of the function, is ununsed.

Reply:
done